src/3p-frame.js

```diff
@@ -135,7 +136,10 @@ export function getIframe(
 export function addDataAndJsonAttributes_(element, attributes) {
   for (let i = 0; i < element.attributes.length; i++) {
     const attr = element.attributes[i];
-    if (attr.name.indexOf('data-') != 0) {
+    if (!startsWith(attr.name, 'data-')
+      // data-vars- is reserved for amp-analytics
+      // see https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/analytics-vars.md#variables-as-data-attribute
+      || startsWith(attr.name, 'data-vars-')) {
       continue;
     }
     attributes[dashToCamelCase(attr.name.substr(5))] = attr.value;
```

Comment:
I commented on another issue in the last year where they didn't use it because they needed the dash-case strings, which seemed fine. But in this case, I think it's the better choice.